website/pages/component/[id]/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { GetStaticPaths, GetStaticProps } from 'next';
+import fs from 'fs';
+import path from 'path';
+import Layout from '../../../components/layout';
+import { BaseLayoutProps, markdownToHtml } from '../../../utils';
+
+export default function Component({
+  component,
+  changelog,
+  readme,
+}: BaseLayoutProps) {
+  return <Layout component={component} changelog={changelog} readme={readme} />;
+}
+
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  if (!params) {
+    return { props: {} };
+  }
+
+  const { id } = params;
+
+  let changelogMarkdown, readme;
+
+  if (typeof id === 'string') {
+    changelogMarkdown = fs.readFileSync(
+      path.join('../packages', id, 'changelog.md'),
+    );
+
+    readme = fs
+      .readFileSync(path.join('../packages', id, 'README.md'))
+      .toString();
+  }
+
+  const changelog = await markdownToHtml(changelogMarkdown || '');
+
+  return {
+    props: {
+      changelog,
+      readme,
+      component: id,
+    },
+  };
+};
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const packages = fs.readdirSync('../packages');
+
+  const paths = packages.map(packageName => ({
```

Comment:
nit, it's the folder name, not the package name. that's defined inside the package.json

```suggestion
  const paths = packages.map(folderName => ({
```

Reply:
👍